src/Core/Addon/Module/ModuleManager.php

```diff
@@ -71,15 +73,24 @@ class ModuleManager implements AddonManagerInterface
      */
     private $translator;
 
+    /**
+     * @var Employee Legacy employee class
+     */
     private $employee;
+    
+    /**
+     * @var EventDispatcherInterface
+     */
+    private $dispatcher;
 
     public function __construct(AdminModuleDataProvider $adminModulesProvider,
         ModuleDataProvider $modulesProvider,
         ModuleDataUpdater $modulesUpdater,
         ModuleRepository $moduleRepository,
         ModuleZipManager $moduleZipManager,
         TranslatorInterface $translator,
-        Employee $employee = null)
+        Employee $employee = null,
+        EventDispatcherInterface $dispatcher = null)
```

Comment:
This could happen when we do not instantiate this class with the service container from Symfony (i.e with `ModuleManagerBuider`:/).

And for the tests, I do not need the dispatcher to test this class. `TabRegister`and `TabUnregister` will have their own tests.